### ISSUE TITLE
Inconsistent behavior regarding loading map files

### ISSUE THREAD

**Issue by yanlee0217** [open-rmf/rmf_site#112]
### Description
When attempting to load map files, I encountered issues regarding loading building yaml files that seem to be incosistent.
I have used demo worlds from open-rmf/rmf_demos. I experienced that the "clinic_world" map file loaded correctly, but the "airport_terminal" map file led to errors. (I can look into other maps later today)

### Expected behavior
All states, goals, levels, etc should be loading seamlessly as there is no change in yaml syntax.

### Actual behavior
Some of the map files, including the "airport_terminal" file, produced error messages when attempting to load them.
**Steps taken**
- Install open-rmf/rmf_site and open-rmf/rmf_demos.
- Attempt to load "clinic_world" with no failure.
- Attempt to load the "airport_terminal" map file in open-rmf/rmf_site.
- Observe the error message: "_invalid type: map, expected unit struct Transition_".
- debug error by commenting out transition and states from building yaml file, observed new error "_invalid type: map, expected unit struct GoalSet_".

I suspect that the issue may be related to the crowd-sim descriptors in the yaml file, as commenting out those sections produced a different error message.
It is unclear to me whether the syntax for the yaml files has changed significantly between rmf_site and previous versions, which could have led to issue observed.
Would also suggest that we catch the error and display it on the GUI, for ease of use. Current behavior only shows up in the terminal.

**Comment by mxgrey** [comment]
Thanks for reporting this!

The site editor is still at an experimental stage, so I'm not surprised that there are some building files which aren't loading correctly yet. It's good to know that the airport terminal world isn't working; I'll be sure to test that out and get it working asap.

I fully agree that errors should be presented in a clear way to the user through the GUI. I've already opened a feature request issue for that [here](https://github.com/open-rmf/rmf_site/issues/71), and I think this error is a situation which can also be covered by that status console feature. Feel free to comment on that issue if you have any further thoughts about how to present errors through the GUI.

I'll leave this issue open until we've fixed the ability to load the airport terminal world.
